main.tf

```diff
@@ -40,6 +52,16 @@ resource "google_storage_bucket" "application" {
   }
 }
 
+# ------------------------------
+# Binding Google Tags to buckets
+# ------------------------------
+resource "google_tags_location_tag_binding" "binding" {
+  for_each  = { for bucket_tag in local.generated_map_bucket_tags : "${bucket_tag.bucket_name}.${bucket_tag.bucket_location}.${bucket_tag.tag_value}" => bucket_tag }
```

Comment:
Just for convention, we use `--` as separator.

Reply:
Switched Separators